internal/server/server.go

```diff
@@ -26,10 +26,41 @@ type Server struct {
 func New(cfg *Config, sqlDb *sql.DB) (*Server, error) {
 	mux := http.NewServeMux()
 
-	mux.HandleFunc("/api/{table}/{relation...}", func(w http.ResponseWriter, r *http.Request) {
-		// Handle tables with a 1-M relation
-		// When relation is accessed, fetch all results FK to the {table} for {relation...}
-		w.Write([]byte("Ok"))
+	mux.HandleFunc("/api/tables", func(w http.ResponseWriter, r *http.Request) {
+		query := "SELECT name, type FROM sqlite_master WHERE type IN ('table', 'view')"
+
+		if r.URL.Query().Has("order") {
+			ordering := r.URL.Query().Get("order")
+			query = fmt.Sprintf("%s ORDER BY `name` %s", query, ordering)
+		}
+
+		rows, err := sqlDb.QueryContext(r.Context(), query)
```

Comment:
Probably overkill but we could look at caching the table listing query response to reduce IO. Though It's unlikely that we would want to optimize for such a small gain.